source/administration/changelog.md

```diff
@@ -2,6 +2,224 @@
+   - Added `"Encrypt": false`to control whether Mattermost will decrypt SAML Assertions encrypted with your Service Provider Public Certificate.
+   - Added `"IdpUrl": ""` to set the SAML SSO URL where Mattermost sends a SAML request to start login sequence.
+   - Added `"IdpDescriptorUrl": ""` to set the Identity Provider Issuer URL for the Identity Provider you use for SAML requests.
+   - Added `"AssertionConsumerServiceURL": ""` to set the Service Provider Login URL.
+   - Added `"IdpCertificateFile": ""` to set the public authentication certificate issued by your Identity Provider.
+   - Added `"PublicCertificateFile": ""` to set certificate used to generate the signature on a SAML request to the Identity Provider for a service provider initiated SAML login, when Mattermost is the Service Provider.
+   - Added `"PrivateKeyFile": ""` to set the private key used to decrypt SAML Assertions from the Identity Provider.
+   - Added `"FirstNameAttribute": ""` to set the attribute in the SAML Assertion that will be used to populate the first name of users in Mattermost.
+   - Added `"LastNameAttribute": ""` to set the attribute in the SAML Assertion that will be used to populate the last name of users in Mattermost.
+   - Added `"EmailAttribute": ""` to set the attribute in the SAML Assertion that will be used to populate the email of users in Mattermost.
+   - Added `"UsernameAttribute": ""` to set the attribute in the SAML Assertion that will be used to populate the username of users in Mattermost.
+   - Added `"NicknameAttribute": ""` to set the attribute in the SAML Assertion that will be used to populate the nickname of users in Mattermost.
+   - Added `"LocaleAttribute": ""` to set the attribute in the SAML Assertion that will be used to populate the language of users in Mattermost.
+   - Added `"LoginButtonText": ""` set the text that appears in the login button on the login page.
+
+#### Database Changes from v3.1 to v3.2
+
+**TeamMembers Table**    
+- Added `DeleteAt` column.
+
+**Emoji Table**    
+- Added `Emoji` table.
+
+### Known Issues
+
+- “More” option under Direct Message list no longer shows count of team members not in your direct message list.
+- On Firefox, search results for hashtags are not properly highlighted.
+- Clicking on a desktop notification from another team doesn’t open the team.
+- Webhook attachments don't show up in search results.
+- On Firefox, System Console sidebar completely disappears when an LDAP setting is saved
+- On Firefox, `CTRL/CMD + U` keyboard shortcut doesn't work
+- Copying and pasting an image from a browser doesn't work
+- `/join` sometimes throws an error
+- When upgrading to 3.X, syntax highlighting using Solarized code theme is lost
+- In Compact view, clicking on a file in the first post in the right hand sidebar attempts to download the file
+- Unable to leave a private channel in mobile view
+- Invalid config setting causes server to panic on start
```

Comment:
I think this is fixed, maybe check with EN https://mattermost.atlassian.net/browse/PLT-3364